README.md

```diff
@@ -2,99 +2,63 @@
-
-1. `script/server --development`
-2. When complete, press Ctrl+C to stop the server (this allows you to start the server in debug mode instead).
-
-You should now be able to run the `App` run group.
-
-### Data import
-
-If you’re running Docker manually (recommended) you will need to enter a Bash shell inside the container, in order to run any of the data import commands:
-
-    docker-compose exec web bash
-
-If you’re running Docker via Visual Studio Code, instead, you’ll want to run the commands via the built-in terminal.
-
-You will likely want to create a Django superuser, by running this inside the container:
-
-    script/createsuperuser
+```bash
+bw login
+bw get attachment .env --itemid 064abbb1-e41b-4632-be90-b2270105d4d1
+```
 
-The superuser will be created with the details specified in the `DJANGO_SUPERUSER_*` environment variables. [Read more about how Docker handles environment variables](https://docs.docker.com/compose/envvars-precedence/).
+2. Open the project folder in VSCode. You should receive a notification in the bottom left to open the project in
+   a Dev Container. If not, press `Cmd+Shift+P` to open the command palette, and type "reopen in container" to find
+   the "Dev Containers: Reopen in Container" option.
 
-You will also want to import data. `import_last_election_data` requires you have the 2019 election results ([download here](https://researchbriefings.files.parliament.uk/documents/CBP-8749/HoC-GE2019-results-by-constituency.csv)), saved as `data/2019_general_election.csv`.
+3. Wait for the Dev Containers to set up. This should take 5-10 minutes.
 
-For convenience, there is a script that will automatically run all the `import_*` commands from `hub/management/commands`, one after the other, which you can run from inside the container:
+4. Make sure the Python Debugger extension is installed by opening the "Extensions" panel on the left hand side and
+   searching for "ms-python.debugpy".
 
-    script/import-all-data
+5. In the "Run and Debug" section of the VSCode left-hand navigation, run the "Run Mapped!" configuration to start the Mapped
+   application.
 
-You could alternatively run commands individually (again, from inside the container), eg:
+6. Visit http://localhost:3000 in your browser to access Mapped.
 
-    ./manage.py import_areas
-    ./manage.py import_mps
+### Feature Showcase
 
-### Logging In
+1. Log in to the Mapped [front end](http://localhost:3000/login). The username and password are `admin` and `password`.
+2. You should have 3 data sources: "Seed Member List", "Seed Custom Data", and "Seed Events".
+3. This data source should have a mapping that populates the Constituency and Element fields in the AirTable.
+4. You should have a map report called "Test map report". Constituencies and MPs should be displayed.
+5. There should be a hub at [http://hub.localhost:3000](http://hub.localhost:3000).
+6. The hub should have some minimal content on the homepage, and a working pledge map.
```

Comment:
Clicking the "pledge to take part" button doesn't work:

<img width="1390" alt="image" src="https://github.com/user-attachments/assets/39720036-5d9d-4d0c-bb12-80a5e51f0df2">

Reply:
This is still an issue

Reply:
I'm going to split this off into a new issue.